pkg/config/constants.go

```diff
@@ -14,6 +14,9 @@ import (
 // ModelType defines a type for model strings.
 type ModelType string
 
+// Store version set during build
+var Version = "Version"
```

Comment:
Versionings are good candidates for having a top level `internal` folder, e.g.  `internal/version/`. `pkg/` signals that can be imported (and since this is a variable, it can be tweaked during runtime).